bundles/org.openhab.binding.renault/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -11,10 +11,19 @@
 
 		<channels>
 			<channel id="batterylevel" typeId="system.battery-level"/>
+			<channel id="batteryavailableenergy" typeId="batteryavailableenergy"/>
+			<channel id="plugstatus" typeId="plugstatus"/>
+			<channel id="chargingstatus" typeId="chargingstatus"/>
+			<channel id="chargingmode" typeId="chargingmode"/>
+			<channel id="chargingremainingtime" typeId="chargingremainingtime"/>
+			<channel id="estimatedrange" typeId="estimatedrange"/>
+			<channel id="odometer" typeId="odometer"/>
 			<channel id="hvacstatus" typeId="hvacstatus"/>
+			<channel id="hvactargettemperature" typeId="hvactargettemperature"/>
+			<channel id="externaltemperature" typeId="externaltemperature"/>
```

Comment:
I think the existing system channel type is appropriate here : system.outdoor-temperature

Reply:
Great idea but...  I want to hide this because it does not work on most cars so I want it to be "Advanced"  (Advanced means it probably will not work great....)

Reply:
Advanced is not for channels that do not work !
Advanced is for channels that are working but only a part of users will need them.
But I agree that the system channel type cannot be advanced, so keep your specific channel type and mention in the channel description that this channel is working only with few car models. To be mentioned in documentation too.

Reply:
The channel works but the api only delivered a value on 1 of the 3 test cars.  I have this sentence in the README.md

"The "Advanced" channels are especially likely to cause confusion and problems."

I want to avoid describing specific problems because I do not know what works where and can not support individual car models...  So I am purposefully vague.  

This is not great but the alternative is to remove these channels from the binding.

What would you advice about the best approach to theses API quirks...

Reply:
A good practice is to add a chapter "Limitations" at the end of the README. You could then remove your sentence from the channels chapter and in this new chapter you explain clearly which channels are working only for some of the cars. I see only two advanced channels, the outside temperature and the target heat temperature.

Reply:
Added text to README.